src/templates/rule.js

```diff
@@ -93,15 +95,15 @@ const Rule = ({ data, location }) => {
     }
   };
 
-  const getRelatedRule = (relatedRuleUri) => {
+  const getRelatedRule = (relateduri) => {
```

Comment:
Why did this change name?

Make the variable name camelCase
```suggestion
  const getRelatedRule = (relatedUri) => {
```